__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require("supertest");
+const app = require("../app.js");
+
```

Comment:
Your test file is missing: 

That the seed is ran before each test (check the promise is returned)
The connection ended in an afterAll

Reply:
done now: https://github.com/hedieh623/Hedieh-s-News/blob/12a26e345bc96160f0a32aadf8bb55a56db206a3/__tests__/app.test.js#L5-L13

thank you!